packages/node_modules/cerebral/src/views/preact/StateContainer.js

```diff
@@ -8,7 +8,7 @@ class StateContainer extends Component {
     }
   }
   render() {
-    return this.props.children
+    return this.props.children[0]
```

Comment:
Btw. Preact Containers don't support more than one child :(
it renders only first children.  you can't use it like 
```
<StateContainer>
    <TestComponent />
    <AnotherComponent />
</StateContainer>
```
It does not render AnotherComponent.